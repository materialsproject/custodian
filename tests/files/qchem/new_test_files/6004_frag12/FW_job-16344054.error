/global/u2/s/sblau/pymatgen/pymatgen/io/cif.py:44: UserWarning: Please install optional dependency pybtex if youwant to extract references from CIF files.
  warnings.warn("Please install optional dependency pybtex if you"
DEBUG:launchpad:FW with id: 6004 is unique!
DEBUG:launchpad:Created/updated Launch with launch_id: 7670
DEBUG:launchpad:RUNNING FW with id: 6004
INFO:rocket.launcher:RUNNING fw_id: 6004 in directory: /global/cscratch1/sd/sblau/ion_placement/block_2018-11-13-02-29-26-951243/launcher_2018-11-14-02-46-59-353432
INFO:rocket.launcher:Task started: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}}.
INFO:rocket.launcher:Task completed: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}} 
INFO:rocket.launcher:Task started: {{atomate.qchem.firetasks.run_calc.RunQChemCustodian}}.
INFO:custodian.custodian:Run started at 2018-11-13 18:56:03.019818 in /global/cscratch1/sd/sblau/ion_placement/block_2018-11-13-02-29-26-951243/launcher_2018-11-14-02-46-59-353432.
INFO:custodian.custodian:Custodian running on Python version 3.6.4 |Anaconda, Inc.| (default, Jan 16 2018, 18:10:19)  [GCC 7.2.0]
INFO:custodian.custodian:Hostname: nid01231, Cluster: unknown
INFO:custodian.custodian:Starting job no. 1 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Starting job no. 2 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Starting job no. 3 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Starting job no. 4 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Starting job no. 5 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Starting job no. 6 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Logging to custodian.json...
INFO:custodian.custodian:Run ended at 2018-11-13 22:54:55.673844.
INFO:custodian.custodian:Run completed. Total time taken = 3:58:52.654026.
