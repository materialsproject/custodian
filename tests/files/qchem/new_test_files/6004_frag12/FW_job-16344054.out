foobar
2018-11-13 18:55:59,196 INFO Hostname/IP lookup (this will take a few seconds)
2018-11-13 18:55:59,204 INFO Launching Rocket
2018-11-13 18:56:02,975 INFO RUNNING fw_id: 6004 in directory: /global/cscratch1/sd/sblau/ion_placement/block_2018-11-13-02-29-26-951243/launcher_2018-11-14-02-46-59-353432
2018-11-13 18:56:02,999 INFO Task started: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}}.
2018-11-13 18:56:03,017 INFO Task completed: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}} 
2018-11-13 18:56:03,019 INFO Task started: {{atomate.qchem.firetasks.run_calc.RunQChemCustodian}}.
